### ISSUE TITLE
ppocrV5文字识别跑了7epoch 怎么  突然acc 变成0 损失也变了，这很奇怪啊，前面一直都正常的

### ISSUE THREAD

**Issue by cqray1990** [PaddlePaddle/PaddleOCR#17848]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

[2026/03/25 06:16:54] ppocr INFO: epoch: [7/75], global_step: 27870, lr: 0.000494, acc: 0.976562, norm_edit_dis: 0.980469, CTCLoss: 0.093047, NRTRLoss: 1.400669, loss: 1.494152, avg_reader_cost: 0.00273 s, avg_batch_cost: 2.52638 s, avg_samples: 320.0, ips: 126.66328 samples/s, eta: 8 days, 17:53:45, max_mem_reserved: 43928 MB, max_mem_allocated: 41308 MB
[2026/03/25 06:17:21] ppocr INFO: epoch: [7/75], global_step: 27880, lr: 0.000494, acc: 0.978125, norm_edit_dis: 0.980859, CTCLoss: 0.098326, NRTRLoss: 1.403208, loss: 1.505783, avg_reader_cost: 0.00287 s, avg_batch_cost: 2.67612 s, avg_samples: 320.0, ips: 119.57593 samples/s, eta: 8 days, 17:53:36, max_mem_reserved: 43928 MB, max_mem_allocated: 41308 MB
[2026/03/25 06:17:48] ppocr INFO: epoch: [7/75], global_step: 27890, lr: 0.000494, acc: 0.981250, norm_edit_dis: 0.982422, CTCLoss: 0.090583, NRTRLoss: 1.412012, loss: 1.518386, avg_reader_cost: 0.00287 s, avg_batch_cost: 2.69142 s, avg_samples: 320.0, ips: 118.89620 samples/s, eta: 8 days, 17:53:28, max_mem_reserved: 43928 MB, max_mem_allocated: 41308 MB
[2026/03/25 06:18:14] ppocr INFO: epoch: [7/75], global_step: 27900, lr: 0.000494, acc: 0.800000, norm_edit_dis: 0.832552, CTCLoss: 1.250202, NRTRLoss: 6.528501, loss: 8.484680, avg_reader_cost: 0.00319 s, avg_batch_cost: 2.58662 s, avg_samples: 320.0, ips: 123.71375 samples/s, eta: 8 days, 17:53:08, max_mem_reserved: 43928 MB, max_mem_allocated: 41308 MB
[2026/03/25 06:18:40] ppocr INFO: epoch: [7/75], global_step: 27910, lr: 0.000494, acc: 0.000000, norm_edit_dis: 0.000000, CTCLoss: 14.098217, NRTRLoss: 6.618148, loss: 20.758375, avg_reader_cost: 0.00327 s, avg_batch_cost: 2.63823 s, avg_samples: 320.0, ips: 121.29346 samples/s, eta: 8 days, 17:52:55, max_mem_reserved: 43928 MB, max_mem_allocated: 41308 MB

### 🏃‍♂️ Environment (运行环境)

paddleocr 3.5
paddlepaddle 3.2.1

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

d

**Comment by scyyh11** [comment]
麻烦提供一下训练配置文件（yaml）、学习率设置、以及数据集大小。这种突然崩塌通常是学习率过大或数据中有异常样本导致的梯度爆炸，看一下配置才能确认。

**Comment by cqray1990** [comment]
> 麻烦提供一下训练配置文件（yaml）、学习率设置、以及数据集大小。这种突然崩塌通常是学习率过大或数据中有异常样本导致的梯度爆炸，看一下配置才能确认。

                                                                       Global:
                                                                    model_name: PP-OCRv5_server_rec # To use static model for inference.
                                                                    debug: false
                                                                    use_gpu: true
                                                                    use_wandb: true
                                                                    epoch_num: 75
                                                                    log_smooth_window: 20
                                                                    print_batch_step: 10
                                                                    save_model_dir: ./output/PP-OCRv5_server_rec_dict_1_c_gene
                                                                    save_epoch_step: 2
                                                                    eval_batch_step: [0, 2000]
                                                                    cal_metric_during_train: true
                                                                    calc_epoch_interval: 1
                                                                    pretrained_model: 
                                                                    checkpoints: 
                                                                    save_inference_dir:
                                                                    use_visualdl: false
                                                                    infer_img: doc/imgs_words/ch/word_1.jpg
                                                                    character_dict_path: ppocr/utils/dict/ppocr_dict_general.txt
                                                                    max_text_length: &max_text_length 198
                                                                    infer_mode: false
                                                                    use_space_char: true
                                                                    distributed: true
                                                                    save_res_path: ./output/rec/ppocr_dict_general.txt
                                                                    d2s_train_image_shape: [3, 48, 320]
                                                                  
                                                                  
                                                                  Optimizer:
                                                                    name: Adam
                                                                    beta1: 0.9
                                                                    beta2: 0.999
                                                                    lr:
                                                                      name: Cosine
                                                                      learning_rate: 0.0005
                                                                      warmup_epoch: 1
                                                                    regularizer:
                                                                      name: L2
                                                                      factor: 3.0e-05
                                                                  
                                                                  
                                                                  Architecture:
                                                                    model_type: rec
                                                                    algorithm: SVTR_HGNet
                                                                    Transform:
                                                                    Backbone:
                                                                      name: PPHGNetV2_B4
                                                                      text_rec: True
                                                                    Head:
                                                                      name: MultiHead
                                                                      head_list:
                                                                        - CTCHead:
                                                                            Neck:
                                                                              name: svtr
                                                                              dims: 120
                                                                              depth: 2
                                                                              hidden_dims: 120
                                                                              kernel_size: [1, 3]
                                                                              use_guide: True
                                                                            Head:
                                                                              fc_decay: 0.00001
                                                                        - NRTRHead:
                                                                            nrtr_dim: 384
                                                                            max_text_length: *max_text_length
                                                                  
                                                                  Loss:
                                                                    name: MultiLoss
                                                                    loss_config_list:
                                                                      - CTCLoss:
                                                                      - NRTRLoss:
                                                                  
                                                                  PostProcess:  
                                                                    name: CTCLabelDecode
                                                                  
                                                                  Metric:
                                                                    name: RecMetric
                                                                    main_indicator: acc
                                                                  
                                                                  Train:
                                                                    dataset:
                                                                      name: LMDBDataSet
                                                                      ds_width: false
                                                                      data_dir: /train
                                                                      ext_op_transform_idx: 1
                                                                      label_file_list:
                                                                      - ./ocr_rec_dataset_examples/trainxx.txt
                                                                      transforms:
                                                                      - DecodeImage:
                                                                          img_mode: BGR   #GRAY
                                                                          channel_first: false
                                                                      - RecAug:
                                                                          tia_prob: 0
                                                                          crop_prob: 0.2
                                                                          reverse_prob: 0.1
                                                                          noise_prob: 0.2
                                                                          jitter_prob: 0.2
                                                                          blur_prob : 0.2
                                                                          hsv_aug_prob : 0.2
                                                                      - MultiLabelEncode:
                                                                          gtc_encode: NRTRLabelEncode
                                                                      - RecResizeImg:
                                                                          image_shape: [ 3, 48, 320 ]
                                                                      - KeepKeys:
                                                                          keep_keys:
                                                                          - image
                                                                          - label_ctc
                                                                          - label_gtc
                                                                          - length
                                                                          - valid_ratio
                                                                  #  sampler:
                                                                  #    name: DefaultSampler
                                                                  #    scales: [[320, 32], [320, 48], [320, 64]]
                                                                  #    first_bs: &bs 128
                                                                  #    fix_bs: false
                                                                  #    divided_factor: [8, 16] # w, h
                                                                  #    is_training: True
                                                                    loader:
                                                                      shuffle: true
                                                                      batch_size_per_card: 328
                                                                      drop_last: true
                                                                      num_workers: 8
                                                                  Eval:
                                                                    dataset:
                                                                      name: LMDBDataSet
                                                                      data_dir: /val
                                                                      label_file_list:
                                                                      - ./ocr_rec_dataset_examples/valxx.txt
                                                                      transforms:
                                                                      - DecodeImage:
                                                                          img_mode: BGR
                                                                          channel_first: false
                                                                      - MultiLabelEncode:
                                                                          gtc_encode: NRTRLabelEncode
                                                                      - RecResizeImg:
                                                                          image_shape: [3, 48, 320]
                                                                      - KeepKeys:
                                                                          keep_keys:
                                                                          - image
                                                                          - label_ctc
                                                                          - label_gtc
                                                                          - length
                                                                          - valid_ratio
                                                                    loader:
                                                                      shuffle: false
                                                                      drop_last: false
                                                                      batch_size_per_card: 128
                                                                      num_workers: 4
                                                                              
                                                                              
                                                                              
@scyyh11
